zammad/values.yaml

```diff
@@ -81,6 +81,13 @@ zammadConfig:
     host: zammad-memcached
     port: 11211
 
+  minio:
+    # enable/disable minio chart dependency
+    enabled: true
```

Comment:
dow we need it by default already?
see question above in chart.yaml.

Reply:
Changed in 6205f2249c6ecfffe6947be13fd23eeb294ee7c6.